content/ch-quantum-hardware/Jaynes-Cummings-model.ipynb

```diff
@@ -66,36 +66,8 @@
     "from qiskit.compiler import transpile, assemble\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        duration = get_closest_multiple_of_16(pulse.seconds_to_samples(readout_time))
</pre><p>You can use new function <code>get_closest_multiple_of</code> you implemented instead of this hard-coding the IBM number.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
OK. I will change .